librosa/display.py

```diff
@@ -386,8 +386,18 @@ def waveplot(y, sr=22050, max_points=5e4, x_axis='time', offset=0.0,
 
         If `None`, no downsampling is performed.
 
-    x_axis : str {'time', 'off', 'none'} or None
-        If 'time', the x-axis is given time tick-marks.
+    x_axis : str or None
+        Display of the x-axis ticks and tick markers. Accepted values are:
+        - 'time' : markers are shown as milliseconds, seconds,
+          minutes, or hours.
+          Values are plotted in units of seconds.
+        - 's' : markers are shown as seconds.
+        - 'ms' : markers are shown as milliseconds.
+        - 'lag' : like time, but past the halfway point counts
+          as negative values.
```

Comment:
I'm not sure that lag formatters make sense for waveplot?  I guess it doesn't hurt to include, but it seems strange to me.

Reply:
that's true. But one application for which it could be useful would be to show window functions.

Reply:
Good point, I hadn't considered that.